etc/kayobe/apt.yml

```diff
@@ -39,12 +47,25 @@
 # * architecture: whitespace-separated list of architectures that will be used
 #   (optional, default is unset)
 # Default is an empty list.
-#apt_repositories:
+apt_repositories:
+  - url: "{{ stackhpc_repo_ubuntu_focal_url }}"
+    suites: "{{ ansible_facts.distribution_release }} {{ ansible_facts.distribution_release }}-updates {{ ansible_facts.distribution_release }}-backports {{ ansible_facts.distribution_release }}-security"
```

Comment:
Wondering why we have security here as well as the security below.

Reply:
My mistake, just a transcription error I think